content/livesync/connector/database.textile

```diff
@@ -106,26 +106,30 @@ GRANT DELETE ON your_nodes_schema.your_nodes_table to your_user;
 
 h2(#outbox). Outbox table
 
-The Database Connector uses the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to send realtime, application-defined change events via Ably's global edge messaging platform. The outbox pattern enables you to reliably publish an ordered sequence of change event messages over Ably, providing exactly-once delivery guarantees.
+The Database Connector utilizes the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to send realtime, application-defined change events via Ably's global edge messaging platform. The outbox pattern allows for the reliable publication of an ordered sequence of change event messages over Ably, providing exact-once delivery guarantees. This pattern involves the use of an outbox table that you create in your database.
```

Comment:
Should be **exactly**-once

Reply:
See 3146122462d451ef6e0069c3a09b11dd3b4a566d